src/CBT.AssemblyInfo/build/Templates/AssemblyInfo.vb

```diff
@@ -7,6 +7,7 @@
 ' </auto-generated>
 '------------------------------------------------------------------------------
 
+Imports System
```

Comment:
it is a bug with VB projects specifically for the SDK projects.  where it doesn't already have a system import and needs it to be specified so the CLRCompliant attribute can be found.